app/src/main/java/ch/epfl/sweng/studyup/utils/Utils.java

```diff
@@ -71,6 +73,7 @@
     public static final String GALLERY = "Gallery";
     public static final String CANCEL = "Cancel";
     public static final String JUSTONCE = "JUST ONCE";
+    public static final String DEFAULT_COURSE_ID = "bVW3QC2SMgLjVRTPLWld";
```

Comment:
Why have something so nasty, why do you need a courses collection on firebase? Why not just put a list of courses in the player collection? Look how we did it for items